src/overmap.cpp

```diff
@@ -5184,6 +5216,7 @@ void overmap::place_cities()
             }
             // Ensure that cities are at least size 2, as city of size 1 is just a crossroad with no buildings at all
             size = std::max( size, 2 );
+            size = std::min( size, 55 );
```

Comment:
Why this restriction?

Reply:
If a city gets too large it's impossible to place.